src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -321,6 +321,14 @@ public static void MapInputTransparent(IViewHandler handler, IView view)
 			if (handler.ContainerView is WrapperView wrapper)
 				wrapper.InputTransparent = view.InputTransparent;
 #else
+
+#if IOS || MACCATALYST
+			// Containers on iOS/Mac Catalyst may be hit testable, so we need to
+			// propagate the the view's values to its container view.
+			if (handler.ContainerView is WrapperView wrapper)
+				wrapper.UpdateInputTransparent(handler, view);
+#endif
```

Comment:
There is a change for iOS to ensure taps pass through the containers if they should.